pkg/ctl/create/nodegroup.go

```diff
@@ -26,6 +26,10 @@ type nodegroupOptions struct {
 
 func createNodeGroupCmd(cmd *cmdutils.Cmd) {
 	createNodeGroupCmdWithRunFunc(cmd, func(cmd *cmdutils.Cmd, ng *api.NodeGroup, options nodegroupOptions) error {
+		if ng.Name != "" && !cmdutils.IsValidNameArg(ng.Name) {
+			return cmdutils.ErrInvalidName(ng.Name)
+		}
+
```

Comment:
does this func ever run for multiple nodegroups? I thought we supported it, but it looks like we only pass in 1 `ng` :thinking:

Reply:
not sure, i thought this runs on a single nodegroup, i'll investigate